cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java

```diff
@@ -103,23 +104,23 @@ public void run_followed_by_show_view(){
         lib.run();
         verify(timerMock).start();
         lib.showView(lib.webView);
-        verify(timerMock).cancel();
+        verify(timerMock , atLeast(1)).cancel();
     }
 
     @Test
     public void run_followed_by_consentFinished(){
         lib.run();
         verify(timerMock).start();
         lib.consentFinished();
-        verify(timerMock).cancel();
+        verify(timerMock, atLeast(1)).cancel();
```

Comment:
we don't need the `atLeast(1)` parameter here.

Reply:
@leodoin yes we need the atleast(1) check because  we are having multiple interaction with counter.cancel(). When webview is open and device loose the internet connection at this time  if user interacts with webview, we are calling first onConsentUIFinished method where we have counter.cancel() call, after that we are getting sendconsent() http call and as we don't have internet connection onErrortask is called this causes another interaction with counter.cancel(). This way we are having multiple interactions with counter.cancel(). This is the reason I am checking atleast(1) interaction.

Reply:
👍